x-pack/plugins/gis/mappings.json

```diff
@@ -12,7 +12,7 @@
       },
       "bounds": {
         "type": "geo_shape",
-        "tree": "quadtree"
+        "strategy": "recursive"
```

Comment:
You can remove this setting as well. In 6.6, geo_shape type was update to be implemented with BKD tree and all of these configuration are no longer needed. https://www.elastic.co/guide/en/elasticsearch/reference/6.x/geo-shape.html

Reply:
Updated